recipes/lifetimes/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "Lifetimes" %}
+{% set version = "0.11.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 75862d86581e75f0c235d830590bea0a9062222f2c9c390949f6432a3fa329b4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - autograd >=1.2.0
+    - dill >=0.2.6
+    - numpy >=1.10.0
+    - pandas >=0.24.0
+    - pip
+    - python
+    - scipy >=1.0.0
+    - setuptools
+  run:
+    - autograd >=1.2.0
+    - dill >=0.2.6
+    - numpy >=1.10.0
+    - pandas >=0.24.0
+    - python
+    - scipy >=1.0.0
+    - setuptools
+
+test:
+  imports:
+    - lifetimes
+    - lifetimes.datasets
+
+about:
+  home: https://github.com/CamDavidsonPilon/lifetimes
+  license: MIT
+  license_family: MIT
+  license_file: README.md
+  summary: Measure customer lifetime value in Python
+  doc_url: https://lifetimes.readthedocs.io/en/latest/?badge=latest
+  dev_url: ''
+
+extra:
+  recipe-maintainers:
+    - CamDavidsonPilon
```

Comment:
@CamDavidsonPilon are you OK being a maintainer?

Reply:
Yup, 👌with me